### DISCUSSION TITLE
Is there a way to delete an artifact associated with an experiment/run

### DISCUSSION THREAD

**Discussion by subhashbylaiah** [mlflow/mlflow#5950] [Q&A] [answered]
Hi,

I am using MLFlow to log artifacts onto a remote MLFlow server. I have a requirement to delete some artifacts associated with a run, without deleting an experiment/run.
I was looking into the MLFlow api and did not see a method for doing that. 

Is there one that I am missing?

Thanks

**Comment by BenWilson2** [comment, marked as answer]
There's definitely not anything that you're missing. 
I'm not entirely sure how common of an operation this is amongst users. 
The mechanism for retrieving artifacts is a directory walk starting at the artifact_uri root for the run. Have you tried manually deleting (or programmatically deleting) the offending artifacts from your artifact store?
